bitwarden/rootfs/etc/services.d/bitwarden/run

```diff
@@ -75,6 +75,11 @@ fi
 export WEBSOCKET_ENABLED=true
 export WEBSOCKET_PORT=8080
 
+# Set domain
+if bashio::config.has_value 'domain'; then
+    export DOMAIN=$(bashio::config 'domain')
```

Comment:
This value can be set in the admin panel, why would this need to be added to the add-on configuration as well?

Reply:
...I did not notice that
Closing